plugins/inventory/nb_inventory.py

```diff
@@ -387,6 +389,7 @@ def _pluralize_group_by(self, group_by):
             "platform": "platforms",
             "device_type": "device_types",
             "manufacturer": "manufacturers",
+            "service": "services",
```

Comment:
No more properties should ever be added here - it's for legacy things like "sites" that didn't make any sense, because "site" is always singular (a NetBox device will only ever have a single site, and I do not forsee that API ever changing)